samples/server/petstore/ruby-sinatra/api/pet_api.rb

```diff
@@ -78,14 +78,14 @@
   "resourcePath" => "/Pet",
   "summary" => "Finds Pets by tags",
   "nickname" => "find_pets_by_tags", 
-  "responseClass" => "array[Pet]", 
+  "responseClass" => "Array&lt;Pet&gt;", 
```

Comment:
Not caused by this PR but maybe worth updating the templates for Sinatra to use {{{ . }}} to avoid HTML escaping.

Reply:
Thanks, I fixed it.